cypress/pageobjects/rancher.po.ts

```diff
@@ -0,0 +1,497 @@
+    private boostrap_page_newPWRepeat = '[style=""] > .labeled-input > input';
+    // private boostrap_page_checkAgreeEULA = '#checkbox-eula > .checkbox-container > .checkbox-custom';
+    private boostrap_page_checkAgreeEULA = '[data-testid="setup-agreement"] > .checkbox-container > .checkbox-custom';
+    private boostrap_page_confirmLogin = '.btn > span';
+    private home_page_mainMenu = '.menu';
+    private home_page_virtualManagement = ':nth-child(7) > .option > div';
+
+    private virtual_page_importButton = '.btn';
+    private virtual_page_clusterName = ':nth-child(1) > .labeled-input > input';
+    private virtual_page_createCluster = '.cru-resource-footer > div > .role-primary';
+
+    private cloudCredential_page_createButton = '.actions > .btn';
+    private cloudCredential_page_harvester = '.subtypes-container > :nth-child(5)';
+    private cloudCredential_page_clusterName = 'input[placeholder="A unique name"]';
+    private cloudCredential_page_confirmCreate = 'button[class="btn role-primary"]';
+    
+    private clusterManagement_page_create = '[href="/dashboard/c/local/manager/provisioning.cattle.io.cluster/create"]';
+    private clusterManagement_rke_selector = '.slider';
+
+    private clusterCreation_page_harvester = ':nth-child(4) > .name';
+
+    private rke2Creation_page_clusterName = 'input[placeholder="A unique name for the cluster"]';
+    private rke2Creation_page_cpus = '[provider="harvester"] > :nth-child(1) > :nth-child(1) > :nth-child(1) > .labeled-input > input';
+    private rke2Creation_page_memory = '[provider="harvester"] > :nth-child(1) > :nth-child(1) > :nth-child(2) > .labeled-input > input';
+    private rke2Creation_page_disk = ':nth-child(1) > :nth-child(2) > :nth-child(1) > .labeled-input > input';
+    private rke2Creation_page_namespaceCombo = '#vs6__combobox';
+    private rke2Creation_page_namespaceOption = '#vs6__option-1';
+    private rke2Creation_page_imageCombo = '#vs7__combobox';
+    private rke2Creation_page_imageOption = '#vs7__option-0';
+    private rke2Creation_page_networkNameCombo = '#vs8__combobox';
+    private rke2Creation_page_networkNameOption = '#vs8__option-0';
+    private rke2Creation_page_ssh_user = 'input[placeholder="e.g. ubuntu"]';
+    private rke2Creation_page_showAdvanced = '.advanced > .hand';
+    private rke2Creation_page_userDataInput = ':nth-child(4) > .yaml-editor > .code-mirror > .vue-codemirror > .CodeMirror > .CodeMirror-scroll > .CodeMirror-sizer > [style="position: relative; top: 0px;"] > .CodeMirror-lines > [style="position: relative; outline: none;"] > .CodeMirror-code > [style="position: relative;"] > .CodeMirror-line';
+    
+    private rke2Creation_page_k8sCombo = '#vs1__combobox';
+    private rke2Creation_page_k8s_rke2Latest = '#vs1__option-1';
+    private rke2Creation_page_k8s_rke2Stable = '#vs1__option-2';
+
+    private rke2Creation_page_k8s_k3sLatest = '#vs1__option-4';
```

Comment:
<img width="530" alt="image" src="https://user-images.githubusercontent.com/18737885/201595330-b779aa5c-e655-4eeb-820f-401faf8d1fe7.png">

`vs1__option-4` will change depending on different environments. In my browser is `vs11__option-4`.

Reply:
Thanks for the instruction, change the menu drop down list selection using `LabeledSelectPo` in `/utils/components/labeled-select.po` 

```
new LabeledSelectPo('.labeled-select', `:contains("Kubernetes Version")`).select({option: rke2_cluster_attributes.rke2_latest, selector: '.vs__dropdown-menu'})
```